### PR TITLE
Add Notification Feature

### PR DESCRIPTION
**Pull request by nauzaf**
7e0be7805f5dfdca3e945d7ce7fde8f7b8c6868b @sendz

### CODE REVIEW

File: assets/custom.js [hunk 1]
```diff
@@ -10,8 +10,20 @@ function activate_sidebar() {
...
-		$('.ui.selection.dropdown').click(function(){
-			$('.ui.dropdown').dropdown('show');
+		$('#project_filter').click(function(){
+			$('#project_filter').dropdown();
 		});		
 	})
-}
\ No newline at end of file
+
+	$(document).ready(function(){
```
Comment (sendz) [inline assets/custom.js]: @fauzan-ngantuk `$(document).ready()` dipanggil sekali aja di satu file js, jangan 3 kali begini...

Comment (sendz) [reply assets/custom.js]: pindahin isinya ke yang atas.

**Review by sendz — CHANGES_REQUESTED** [review]
rapihin custom.js, cuma boleh ada satu `$(document).ready()`.

**Comment by nauzaf** [comment]
@sendz kayak gini bukan --- > 5c5948f81f99f4e4e79a4a814c56276745368136

**Comment by nauzaf** [comment]
@sendz kayak gini ? ----->5c5948f81f99f4e4e79a4a814c56276745368136

File: assets/custom.js (hunk 1 above)
Comment (nauzaf) [reply assets/custom.js]: @sendz kayak gini bukan ? ---- >5c5948f81f99f4e4e79a4a814c56276745368136

**Comment by nauzaf** [comment]
5c5948f81f99f4e4e79a4a814c56276745368136

**Comment by sendz** [comment]
@fauzan-ngantuk 
kenapa nyimpen `$(window).ready()` di dalam function? kalau function itu mau dipanggil sama element lain gak perlu tambahin `$(window).ready()`.

`$(window).ready()` cuma dipake sekali buat script yang dibutuhkan tapi gak akan dipanggil di external, cuma buat inisialisasi aja.

File: assets/custom.js [hunk 2]
```diff
@@ -10,8 +10,16 @@ function activate_sidebar() {
 function activate_dropdown() {
```
Comment (sendz) [inline assets/custom.js]: this implementation is so wrong, bro... if you want to declare a function, don't trigger it when window is ready. instead, if you want the code triggered when your window is ready, don't declare it as a function.

File: assets/custom.js [hunk 3]
```diff
@@ -10,8 +10,16 @@ function activate_sidebar() {
...
 	// body...
 	$(document).ready(function(){
-		$('.ui.selection.dropdown').click(function(){
-			$('.ui.dropdown').dropdown('show');
+		$('#project_filter').click(function(){
```
Comment (sendz) [inline assets/custom.js]: and, you don't need to use `click` listener here as the code below `$('#project_filter').dropdown();` means element with id `project_filter` will have `dropdown` 'feature' whenever the page is loaded.

File: assets/custom.js (hunk 2 above)
Comment (sendz) [reply assets/custom.js]: what I understand from your code is, you are trying to assign any object to have dropdown, so remove function declaration lines instead.

File: assets/custom.js [hunk 4]
```diff
@@ -10,8 +10,16 @@ function activate_sidebar() {
...
+			$('#project_filter').dropdown();
+		});
+
+		$('#feed1').click(function(){
+			$('#feed1').dropdown();
```
Comment (sendz) [inline assets/custom.js]: add a class to any object which decided to have dropdown and initiate your dropdown using a class.

example `$('.with-dropdown').dropdown()` so any classes with `.with-dropdown` will have dropdown feature.

File: assets/custom.js (hunk 2 above)
Comment (sendz) [reply assets/custom.js]: simplify your code to this one

```
$(document).ready(function(){
	$('.ui.sidebar').sidebar('toggle');
	$('.dropdown').dropdown();
});
```

add class `dropdown` to `#project_filter`, `#feed1` and `#feed2`

**Comment by nauzaf** [comment]
@sendz maybe i have to learn the concept first , itu kemaren hasil dari coba-coba awkawk

**Pull request closed without merging**